<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalVariableDatabase</class>
 <widget class="QWidget" name="GlobalVariableDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="CustomGlobalVariableBoolPageWidget" name="m_pBools" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>311</width>
     <height>132</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomGlobalVariableIntPageWidget" name="m_pInts" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>311</width>
     <height>132</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomGlobalVariableFloatPageWidget" name="m_pFloats" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>311</width>
     <height>132</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomGlobalVariableStringPageWidget" name="m_pStrings" native="true">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>311</width>
     <height>132</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>bools</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>integers:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>floats:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>strings:</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomGlobalVariableBoolPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariableboolpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomGlobalVariableIntPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariableintpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomGlobalVariableFloatPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariablefloatpagewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CustomGlobalVariableStringPageWidget</class>
   <extends>QWidget</extends>
   <header>customvariablestringpagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
